controllers/network/ipset_controller.go

```diff
@@ -274,6 +278,13 @@ func (r *IPSetReconciler) reconcileNormal(ctx context.Context, instance *network
 			return ctrl.Result{}, err
 		}
 
+		// sort instance.Status.Reservations by Network
+		sort.Slice(instance.Status.Reservation, func(i, j int) bool {
+			return (strings.EqualFold(instance.Status.Reservation[i].ServiceNetwork,
```

Comment:
in existing env there is no ServiceNetwork. we probably have to fallback in the ipset controller to the network name if thats the case

Reply:
ack, done!